packages/toolpad-components/src/Text.tsx

```diff
@@ -21,31 +21,53 @@ interface TextProps extends Omit<BaseProps, 'children'> {
   loading?: boolean;
 }
 
-const gutters = (marginTop: number, marginBottom: number) => ({
-  marginTop,
-  marginBottom,
-  '&:first-child': {
-    marginTop: 0,
-  },
-  '&:last-child': {
-    marginBottom: 0,
-  },
-});
-
 const MarkdownContainer = styled('div')(({ theme }) => ({
   display: 'block',
   maxWidth: '100%',
   '&:empty::before, & > span:empty::before': {
     content: '""',
     display: 'inline-block',
   },
-  '& h1': { ...theme.typography.h1, ...gutters(16, 16) },
-  '& h2': { ...theme.typography.h2, ...gutters(12, 12) },
-  '& h3': { ...theme.typography.h3, ...gutters(12, 12) },
-  '& h4': { ...theme.typography.h4, ...gutters(12, 12) },
-  '& h5': { ...theme.typography.h5, ...gutters(4, 4) },
-  '& h6': { ...theme.typography.h6, ...gutters(4, 4) },
-  '& p': { margin: 0, marginBottom: 6 },
+  '& h1': {
+    ...theme.typography.h1,
+    marginTop: 16,
+    marginBottom: 16,
+  },
+  '& h2': {
+    ...theme.typography.h2,
+    marginTop: 12,
+    marginBottom: 12,
```

Comment:
I think that the values should be uneven, we almost always need more margin-top than margin-bottom. For example:

```suggestion
    marginTop: 16,
    marginBottom: 12,
```

(I have no idea if this is the exact value that we want, this would be more for **design** to rule this).

Reply:
> this would be for design to define them

Agreed, I treated these mostly as placeholders until design gets a chance to take decisions on how it should look.